eth/v1alpha1/beacon_chain.proto

```diff
@@ -246,6 +246,18 @@ message GetValidatorsRequest {
         // Optional criteria to retrieve the genesis set of validators.
         bool genesis = 2;
     }
+
+    // The maximum number of Validators to return in the response.
+    // This field is optional.
+    uint64 page_size = 3;
+
+    // A pagination token returned from a previous call to `GetValidators`
+    // that indicates where this listing should continue from.
+    // This field is optional.
+    uint64 page_token = 4;
+
+    // Total count of validators.
+    uint64 total_size = 5;
```

Comment:
```suggestion
    int32 total_size = 5;
```
To conform with https://cloud.google.com/apis/design/design_patterns#list_pagination